components/src/atoms/link/Link.js

```diff
@@ -4,14 +4,6 @@ export default {
       type: String,
       required: true
     },
-    inverted: {
-      type: Boolean,
-      default: false
-    },
-    secondary: {
-      type: Boolean,
-      default: false
-    },
```

Comment:
you can remove all props from link component and use in directly as attr on component 
`<a-link href="...`